src/api/app/views/webui/users/notifications/_notifications_list.html.haml

```diff
@@ -9,37 +9,49 @@
           Mark notifications as "Read" and they'll appear here
         - else
           There are no notifications, but there's a world of opportunities!
-    - else
-      .text-center
-        %span.ml-3= page_entries_info notifications, entry_name: 'notification'
-        = link_to_all unless notifications.total_pages == 1 && params[:show_all].nil?
+- else
+  - update_path = my_notification_path(type: selected_filter[:type], project: selected_filter[:project],
+                                       page: params[:page], show_all: params[:show_all])
+  = form_tag(update_path, method: :put, remote: true) do
+    = render partial: 'notification_action_bar', locals: { type: selected_filter[:type] }
+    .card
+      .card-body
+        .text-center
+          %span.ml-3= page_entries_info notifications, entry_name: 'notification'
+          = link_to_all unless notifications.total_pages == 1 && params['show_all'].nil?
+        .list-group.list-group-flush.mt-3
+          - notifications.each do |n|
+            - notification = NotificationPresenter.new(n)
+            .list-group-item.px-0.px-md-2.notifications-grid-container
+              .notifiable-checkbox.mr-2.custom-control.custom-checkbox
+                = check_box_tag('notification_ids[]', notification.id, false,
+                  id: "notification_ids_#{notification.id}", class: 'custom-control-input')
+                = label_tag("notification_ids_#{notification.id}", '', class: 'custom-control-label')
+              .notifiable
+                - if notification.notifiable_type == 'BsRequest'
+                  = image_tag('icons/request-icon.svg', height: 18, title: 'Request notification')
+                  = link_to(notification.notifiable_link[:text], notification.notifiable_link[:path], class: 'mx-1 text-word-break-all')
+                  %span.badge{ class: "badge badge-#{request_badge_color(notification.notifiable.state)}" }
+                    = notification.notifiable.state
+                - else
+                  %i.fas.fa-comments{ title: 'Comment notification' }
+                  = link_to(notification.notifiable_link[:text], notification.notifiable_link[:path], class: 'mx-1 text-word-break-all')
+              .action.ml-auto.align-self-end.align-self-md-start
+                - title, icon = notification.unread? ? ['Mark as "Read"', 'fa-check'] : ['Mark as "Unread"', 'fa-undo']
```

Comment:
I think you can use [this helper](https://github.com/openSUSE/open-build-service/pull/10220/files#diff-249c5d6caddd60654f20780971650cecR53).

Reply:
This button and the one rendered by the helper are a bit different. This button only has an icon while the helper button has some text. There's also an extra class (`px-3`) on the `link_to`.

The helper could be adapted for sure, this just has to be kept in mind.

Reply:
but this reminds me, that I have to adapt the helper a bit, since the notification#update method changed a bit